app/controllers/spree/users_controller_decorator.rb

```diff
@@ -1,3 +1,21 @@
 Spree::UsersController.class_eval do
   layout 'darkswarm'
+
+  prepend_before_filter :load_object, :only => [:edit_address, :update_address, :show, :edit, :update]
+
+  def edit_address
+    country_id = Spree::Config[:default_country_id]
+    @user.build_ship_address(country_id: country_id) unless @user.ship_address
+    @user.build_bill_address(country_id: country_id) unless @user.bill_address
+
+    render :edit_address
+  end
+
+  def update_address
+    if @user.update_attributes(params[:user])
+      redirect_to spree.account_url, :notice => t(:account_updated)
```

Comment:
Old Ruby syntax here as well. And did you try the relative translation key `.account_updated` here? I'm not sure how it works with notices. I sometimes use the relative key and then look at the error message in the front end: "translation key ... missing". So I know where Rails tries to find the translation.